app/javascript/app/components/stories/stories-styles.scss

```diff
@@ -0,0 +1,114 @@
+@import '~styles/settings.scss';
+@import '~styles/layout.scss';
+
+.wrapper {
+  @include row();
+
+  flex-direction: column;
+  height: 1400px;
+
+  @media #{$tablet-portrait} {
+    height: 1080px;
+  }
+
+  @media #{$tablet-landscape} {
+    height: 800px;
+  }
+}
+
+.grid {
+  flex: 1;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas:
```

Comment:
This is cool, although we were avoiding the use of grid because of the problems with IE compatibility. We may have to discuss this

Reply:
I'm trying to make it work on IE according to this article: https://medium.com/@elad/supporting-css-grid-in-internet-explorer-b38669e75d66
I'll add another commit with that